raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
hat could be taken to avoid, mitigate or remedy any adverse effect that may occur as a result of the implementation of the policy, Bill, legislation, strategy, programme, or plan; and [e] any other information that the Minister may by regulations prescribe. [4] Where a person responsible for the Bill, regulation, public policy, strategy, programme, or plan, does not require a Strategic Environmental Assessment under this section, he shall submit a draft of the relevant document to the Minister who shall, as soon as practicable, determine whether or not an assessment is required and shall furnish his decision and the reasons for it. [5] Upon receipt of the strategic environmental assessment report, the Minister shall direct the Director of Environment to review the policy, Bill, legislation, strategy, programme or plan and furnish his opinion to the Minister who shall thereafter give his opinion to the person concerned. [6] A person to whom the Minister has given opinion shall be required to comply with recommendations made and shall be required to submit a fresh report to him indicating: [a] the revisions that have been made to the original document in order to promote environmental management and the sustainable management of natural resources or to avoid, mitigate or remedy any adverse effects which implementation of the original legislation, policy, programme or plan, may have had; [b] any other measures that have to be taken to avoid, mitigate or remedy any adverse effects on the environment and when these were or will be taken, and if any measures recommended by the assessment report have or will not be taken, the reasons for not doing so; and [c] a revised version of the policy, Bill, legislation, strategy, programme or plan. [7] Where the Minister is of the opinion that the environmental concerns raised during the strategic environmental assessment process are not adequately addressed by the revised bill, regulation, policy, strategy, programme or plan, and that additional cost-effective measures to avoid or mitigate against these adverse effects should be taken, he shall within thirty days of receipt of the documents referred to in subsection [5], lodge an objection with the person concerned with a view to reaching agreement on the amendments to be made to the revised policy, Bill, legislation, strategy, programme or plan in order to give effect to the purpose and principles of this Act. [8] Where, upon direction by the Minister, the Director of Environment and the person concerned are unable to reach agreement on the amendments to be made to the revised policy, Bill, legislation, strategy, programme or plan on the measures to be taken, the Director of Environment or the person concerned may lodge a notice of objection to the Minister. [9] The Minister may, upon receipt of the notice of objection, order the documents referred to in subsection [6] to be subjected to public review or to a public hearing before making a final determination. 
```

Comment:
Please replace:
* `likely effects of such regulations` with `likely effects such regulations`; and
* `policy Bills` with `policy, Bill`.